index.js

```diff
@@ -48,96 +48,127 @@ function fromFile(file, callback) {
     });
 }
 
+
+function returnOutput(output) {
```

Comment:
Now that this function is running first with the reordering, mind renaming this function to something like `checkGzip()` or something, so that the purpose of the funciton is clear?